src/components/what-we-do.jsx

```diff
@@ -1,38 +1,133 @@
+  IconClipboardCopy,
+  IconFileBroken,
+  IconSignature,
+  IconTableColumn,
+} from "@tabler/icons-react";
+import { motion, useScroll, useInView } from "framer-motion"
+import Section from "@/components/section-framer";
+
 
 export function WhatWeDoSection() {
+
+  const { scrollYProgress } = useScroll();
+
   return (
-    <Carousel
-      opts={{
-        align: "start",
-        loop: true,
-      }}
-      className="w-full bg-secondary-800 text-primary-500"
+    <motion.div
+      initial={{ opacity: 0, y: 20 }}
+      animate={{ opacity: 1, y: 0 }}
+      transition={{ delay: 1 }}
     >
-      <CarouselContent>
-        {Array.from({ length: 5 }).map((_, index) => (
-          <CarouselItem key={index} className="md:basis-1/2 lg:basis-1/3 bg-secondary-800">
-            <div className="p-1">
-              <Card className="bg-secondary-800 border-s border-secondary-300">
-                <CardContent className="flex aspect-square items-center justify-center p-6">
-                  <span className="text-3xl font-semibold">{index + 1}</span>
-                </CardContent>
-              </Card>
-            </div>
-          </CarouselItem>
-        ))}
-      </CarouselContent>
-      {/* <CarouselPrevious /> */}
-      {/* <CarouselNext /> */}
-    </Carousel>
+      <Section>
+        <div className="flex flex-col p-2 m-2 pb-10">
+
+          <motion.h1
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{ delay: 1 }}
+            className={`md:text-lg text-center text-xs font-medium md:font-light text-secondary-800/50 p-9 mx-auto ${scrollYProgress}`}>OUR ALUMNI'S ARE CURRENTLY AT</motion.h1>
+
+          <div className="h-fit flex flex-col antialiased bg-white dark:bg-black dark:bg-grid-white/[0.05] items-center justify-center relative overflow-hidden ring-1 ring-secondary-400/50 rounded-full md:p-5 md:w-10/12 w-11/12 mx-auto">
+            <SideScroll
+              items={press}
+              direction="left"
+              speed="normal"
+            />
+          </div>
+          <motion.h1
+            initial={{ opacity: 0, y: 20 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={{ delay: 1 }}
+            className=" text-secondary-800 font-semibold text-center text-2xl md:text-4xl p-9 mt-[90px] mb-10 mx-auto">
+            Things we do in <span className="text-primary-400">WebClub🕸️</span>
+          </motion.h1>
+
+
+          <BentoGrids />
+
+
+
+
+
+
+
+        </div>
+
+      </Section>
+
+    </motion.div>
+
   )
 }
+
+
+const press = [
+  "TheNewYorkTimes",
+  "TheWashingtonPost",
+  "Forbes",
+  "Bloomberg",
+  "BusinessInsider",
+  "TechCrunch",
+  "TheGuardian",
+  "Wired",
+];
+
+
+
+export function BentoGrids() {
+  return (
+    <BentoGrid className="max-w-7xl mx-auto md:auto-rows-[20rem]">
+      {items.map((item, i) => (
+        <BentoGridItem
+          key={i}
+          title={item.title}
```

Comment:
A background picture(an illustration representing the what we do) with low opacity for each grid might look good